app/views/settings_dialog.py

```diff
@@ -686,6 +688,50 @@ def _do_todds_tab(self) -> None:
         )
         group_layout.addWidget(self.todds_overwrite_checkbox)
 
+    def _do_themes_tab(self) -> None:
+        tab = QWidget()
+        self.tab_widget.addTab(tab, "Themes")
+
+        tab_layout = QVBoxLayout(tab)
+        tab_layout.setAlignment(Qt.AlignmentFlag.AlignTop)
+
+        group_box = QGroupBox()
+        tab_layout.addWidget(group_box)
+
+        group_layout = QVBoxLayout()
+        group_box.setLayout(group_layout)
+
+        self.themes_label = QLabel("Themes Support")
+        self.themes_label.setFont(GUIInfo().emphasis_font)
+        group_layout.addWidget(self.themes_label)
+
+        self.enable_themes_checkbox = QCheckBox(
+            "Enable to use .qss Themes (stylesheet) / Disable to use OS/System Theme"
+        )
+        group_layout.addWidget(self.enable_themes_checkbox)
+
+        self.themes_combobox = QComboBox()
+        self.themes_combobox.setSizePolicy(
+            QSizePolicy.Policy.Maximum, QSizePolicy.Policy.Preferred
+        )
+
+        # Get the available themes from the Themes class
+        available_themes = [folder.name for folder in Themes.get_available_themes()]
+        self.themes_combobox.addItems(available_themes)
+        group_layout.addWidget(self.themes_combobox)
+
+        # Add Theme info label
+        self.themes_info_label = QLabel(
+            "How to add your own Themes (stylesheet) \n"
+            "1) Create a new folder in 'themes' folder in your RimSort Config Folder \n"
+            "2) Using the default 'RimPy' Theme As Refrence create your own 'style.qss' file in the folder you created \n"
+            "3) Start RimSort and Select your theme from the Dropdown \n"
+            "4) Restart RimSort to apply the Theme \n\n"
+            "Note: The Name of Folder will be used as Name of the Theme / Invalid Themes will be ignored"
```

Comment:
Fix spelling mistakes and inconsistent capitalization.

Also, I think this info would be better in its own dialogue instead of a QLabel. Then you get rich text formatting and don't have everything stuck as QLabel stylization which is just bold.

Reply:
Additionally, should add a button to open the theme directory in explorer.

Reply:
Done